service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/DeviceConnectionState.java

```diff
@@ -11,7 +11,11 @@
 public enum DeviceConnectionState
 {
     Disconnected("Disconnected"),
-    Connected("Connected");
+    Connected("Connected"),
+
+    // For some reason, preview hubs send this value with a lower case "disconnected" vs "Disconnected" and "connected" vs "Connected"
```

Comment:
We won't keep this when we merge. This is just a workaround for a bug that appears to only be present in preview hubs